salt/config/__init__.py

```diff
@@ -1578,6 +1586,8 @@ def _gather_buffer_space():
     'winrepo_passphrase': '',
     'winrepo_refspecs': _DFLT_REFSPECS,
     'syndic_wait': 5,
+    'jinja_line_statement_prefix': None,
```

Comment:
Possibly, I went for the str & None combo because it seemed to make more sense logically to me but also because there were plenty of other options also using str & None in the same way. For example:

- environment L229 & L1399
- state_top_saltenv L240 & L1512
- log_level_logfile L373 & L1530
- etc..

But I do agree that `''` should fit these two options better, plus I can remove the (hackish?) explicit check for `'none'`.

Thanks, will update with `''` as the default.